views/update/update-check-your-answers.html

```diff
@@ -19,6 +19,20 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application
       <br>
       {% include "includes/entity.html" %}
       <br>
+      {% if (appData.beneficial_owners_individual and appData.beneficial_owners_individual.length > 0)
```

Comment:
Don't you need to check that ch_reference is defined to only show reviewed BOs?

Reply:
Yep, just waiting on the pr that adds ch_reference I believe, working on how to filter based on any attribute for the time being.